### ISSUE TITLE
[Feature Request] Provide an API for directory specific sidebars

### ISSUE THREAD

**Issue by avik-pal** [LuxDL/DocumenterVitepress.jl#70]
Vitepress supports this https://vitepress.dev/reference/default-theme-sidebar#multiple-sidebars so it might be good to support that.

Currently, for long documentations, the sidebar is quite big

![image](https://github.com/LuxDL/DocumenterVitepress.jl/assets/30564094/645c1922-74ad-4fd8-83f9-dc1fc8facc2d)

**Comment by asinghvi17** [comment]
Would it be better to have them collapse instead?  I could probably add that behaviour in relatively easily...

**Comment by avik-pal** [comment]
Yes, collapsible is a good way too. I tried to look for an option where only the current page sidebar is expanded, but Vitepress doesn't provide that (unless I missed something). For example, if I am in "Overview" then "Getting Started" is expanded but all other sections are collapsed.

**Comment by lazarusA** [comment]
<img width="273" alt="Screenshot 2024-03-12 at 21 09 03" src="https://github.com/LuxDL/DocumenterVitepress.jl/assets/19525261/7f0b98f6-831a-434f-aa22-e5bca3106324">

if you set everything to collapse, then the one that active will be open, and the others hidden.

**Comment by avik-pal** [comment]
That's perfect.
